sdk/devcenter/azure-developer-devcenter/src/main/java/com/azure/developer/devcenter/DevBoxesClient.java

```diff
@@ -0,0 +1,674 @@
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
+    public SyncPoller<BinaryData, BinaryData> beginCreateDevBox(
+            String projectName, String userId, String devBoxName, BinaryData body, RequestOptions requestOptions) {
+        return this.client.beginCreateDevBox(projectName, userId, devBoxName, body, requestOptions).getSyncPoller();
+    }
+
+    /**
+     * Deletes a Dev Box.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * {
+     *     name: String (Optional)
+     *     projectName: String (Optional)
+     *     poolName: String (Required)
+     *     provisioningState: String (Optional)
+     *     actionState: String (Optional)
+     *     powerState: String(Unknown/Deallocated/PoweredOff/Running/Hibernated) (Optional)
+     *     uniqueId: String (Optional)
+     *     errorDetails (Optional): {
+     *         code: String (Optional)
+     *         message: String (Optional)
+     *     }
+     *     location: String (Optional)
+     *     osType: String(Windows) (Optional)
+     *     user: String (Optional)
+     *     hardwareProfile (Optional): {
+     *         skuName: String (Optional)
+     *         vCPUs: Integer (Optional)
+     *         memoryGB: Integer (Optional)
+     *     }
+     *     storageProfile (Optional): {
+     *         osDisk (Optional): {
+     *             diskSizeGB: Integer (Optional)
+     *         }
+     *     }
+     *     imageReference (Optional): {
+     *         name: String (Optional)
+     *         version: String (Optional)
+     *         operatingSystem: String (Optional)
+     *         osBuildNumber: String (Optional)
+     *         publishedDate: OffsetDateTime (Optional)
+     *     }
+     *     createdTime: OffsetDateTime (Optional)
+     *     localAdministrator: String(Enabled/Disabled) (Optional)
+     * }
+     * }</pre>
+     *
+     * @param projectName The DevCenter Project upon which to execute operations.
+     * @param userId The AAD object id of the user. If value is 'me', the identity is taken from the authentication
+     *     context.
+     * @param devBoxName The name of a Dev Box.
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @return the {@link SyncPoller} for polling of a DevBox Dev Box.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
+    public SyncPoller<BinaryData, BinaryData> beginDeleteDevBox(
```

Comment:
codegen need to think about whether we should use `SyncPoller<BinaryData, Void>` for LRO delete.

@alzimmermsft Do you have opinion (as Srikanta is OOF)? Or anyone else have input?

Reply:
Based on the Swagger the DevBox delete returns a DevBox, so to me this should at least be `SyncPoller<Void, BinaryData>` (`SyncPoller<PollResponse, FinalResponse>`).

Reply:
@alzimmermsft -- this was unfortunately a response type in our spec that was required by the API review board; they claimed it was necessary to avoid some issues with code gen. It is not actually a valid response from our API; we will only ever return a 202 or 204 as a successful response, and will never have a return.

Reply:
Thanks Alan and Chris. Yeah, the definition in swagger for LRO is kind of guesswork for codegen. Cadl should be much better on the polling response type, but might still not great on final response type.

On Chris description, I take the DELETE to an existing resource would be (if none exist, it will be just 204 NoContent?),
```
DELETE -> 202 status=InProgress
GET -> 200 status=InProgress
GET -> 200 status=InProgress
GET -> 200 status=Succeeded
```
And there will be no more requests.

If this case, it should be `SyncPoller<BinaryData, Void>`? (and yes, `SyncPoller<BinaryData, BinaryData>` is the safest and for that reason we do this in current codegen)

Here we kind of guessing DELETE LRO (if done properly in backend) will never have a final result type?

Reply:
Yup that's exactly the correct request flow!

Reply:
Sure. I will keep this PR as it is. And I will prepare a codegen which would output `SyncPoller<BinaryData, Void>` for DELETE LRO. And I will see if I use that one to make a folllow-up PR, after this get merged.